packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -209,13 +232,13 @@ export class DbSyncStakePoolProvider extends DbSyncProvider(RunnableModule) impl
         this.getPoolsDataOrdered(poolUpdates, totalAdaAmount, options)
       );
     // Get stake pools APYs cached
-    const poolAPYs =
-      sortType === 'apy'
+    const poolAPYs = this.response?.search?.metrics?.apy
+      ? sortType === 'apy'
         ? (orderedResult as PoolAPY[])
```

Comment:
Lint error here for nested ternary

Reply:
Ah yes, I was going to circle back to this. We've actually got a conflict between prettier rules and something else in the eslint presets, as we can use the syntax with parentheses, but prettier strips them :upside_down_face:

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/609/commits/326c8369a478502f00dddd2dd6f0efdc0dad919b

Reply:
I've ran into it in a few occasions as well :sweat_smile: